model-optimizer/mo/ops/softmax.py

```diff
@@ -67,15 +67,23 @@ class LogSoftmax(Op):
 
     def __init__(self, graph: Graph, attrs: dict):
         super().__init__(graph, {
-            'infer': None,
-            'kind': 'op',
-            'axis': 1,
-            'type': None,  # the operation will be replaced with a x - Log(ReduceSum(Exp(x), axis)) sub-graph
+            'type': __class__.op,
             'op': __class__.op,
+            'version': 'opset5',
+            'infer': LogSoftmax.infer,
+            'axis': -1,
             'in_ports_count': 1,
             'out_ports_count': 1,
         }, attrs)
 
+    @staticmethod
+    def infer(node: Node):
+        if node.axis < 0:
```

Comment:
Was it specified as attribute? not a input?

Reply:
As attribute